### PR TITLE
life is better when you don't block requests on disk io

### PR DESCRIPTION
**Pull request by sandfox**
yo @MicheleBertoli

### CODE REVIEW

**Comment by MicheleBertoli** [comment]
Thanks @sandfox <3

File: server.js [hunk 1]
```diff
@@ -30,9 +32,7 @@ app.get('*', (req, res) => {
     fetchData(state).then((data) => {
       const props = {data, state, req};
       const html = React.renderToString(<Root {...props} />);
-      let template = fs.readFileSync(path).toString();
-      template = template.replace('__HTML__', html);
-      template = template.replace('__DATA__', JSON.stringify(data));
+      const content = template.replace('__HTML__', html).replace('__DATA__', JSON.stringify(data));
       res.send(template);
```
Comment (MicheleBertoli) [inline server.js]: We should send `content`, right?

Comment (MicheleBertoli) [inline server.js]: We should send `content`, right?

File: server.js [hunk 2]
```diff
@@ -30,9 +32,7 @@ app.get('*', (req, res) => {
     fetchData(state).then((data) => {
       const props = {data, state, req};
       const html = React.renderToString(<Root {...props} />);
-      let template = fs.readFileSync(path).toString();
-      template = template.replace('__HTML__', html);
-      template = template.replace('__DATA__', JSON.stringify(data));
+      const content = template.replace('__HTML__', html).replace('__DATA__', JSON.stringify(data));
```
Comment (MicheleBertoli) [inline server.js]: If you just could split these lines, it would be great :)

Comment (sandfox) [inline server.js]: do you want to assign the output of the  first `replace` to a temporary variable? or happy with just new lining it?

```
let content = template.replace('__HTML__', html);
content =  content.replace('__DATA__', JSON.stringify(data));
```

or

```
const content = template.replace('__HTML__', html)
  .replace('__DATA__', JSON.stringify(data));
```

**Comment by sandfox** [comment]
@MicheleBertoli made those changes (I think)

**Comment by MicheleBertoli** [comment]
Amazing, merged ;)

**Pull request merged**